js/src/settings.js

```diff
@@ -89,7 +89,9 @@
     ],
 
     'availableAnnotationDrawingTools': [
-
+      /** possible drawing tools:
+       * 'Rectangle', 'Ellipse', 'Freehand', 'Polygon', 'Pin'
+       **/
```

Comment:
I think the default here should probably not be an empty array.  We could probably set it to all of them, and then a developer can choose to narrow down the options from there. @aeschylus what do you think?